### PR TITLE
Add a "Reviewed by Hound" badge

### PR DESCRIPTION
**Pull request by salbertson**
What do you think about adding a Hound badge?

screenshot 2018-12-09 18 12 29

### CODE REVIEW

**Comment by vesan** [comment]
Why would we? We are not using Hound anywhere, right?

**Comment by salbertson** [comment]
Hey @vesan, it looks like it. 😄 

screenshot 2018-12-10 09 31 50

**Comment by germ2** [comment]
voice input could be a nice addition

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Rails Girls Guides
```
Comment (germ2) [inline README.md]: maybe try girls streamline guides? im new and don't even know if I should be making suggestions here

**Comment by hanislong** [comment]
Gooooodd

**Comment by coolbrg** [comment]
Same question as @vesan asked

> Why would we? We are not using Hound anywhere, right?

🤔 

It seems there is no CI file in the repo but somehow we are getting HoundCI checks and I am just wondering how. :)

**Comment by Djritz50** [comment]
9boy

**Pull request closed without merging**